### ISSUE TITLE
Several false positives

### ISSUE THREAD

**Issue by Frulfump** [dotnet-project-file-analyzers/dotnet-project-file-analyzers#757]
I get the warnings below for this project

```xml
<Project Sdk="Microsoft.NET.Sdk">

  <PropertyGroup>
    <OutputType>Exe</OutputType>
    <TargetFramework>net10.0</TargetFramework>
  </PropertyGroup>

  <ItemGroup>
    <PackageReference Include="BenchmarkDotNet" />
  </ItemGroup>

  <ItemGroup>
    <ProjectReference Include="../x.csproj" />
  </ItemGroup>

</Project>
```

And x.csproj is

```xml
<Project Sdk="Microsoft.NET.Sdk">
  <PropertyGroup>
    <OutputType>Exe</OutputType>
    <TargetFramework>net10.0</TargetFramework>
  </PropertyGroup>

</Project>
```

And a `Directory.Build.props` with just basic stuff 

Proj0200: Define the <IsPackable> node explicitly (https://dotnet-project-file-analyzers.github.io/rules/Proj0200.html)
Proj0400: Define the <IsPublishable> node explicitly (https://dotnet-project-file-analyzers.github.io/rules/Proj0400.html)
Proj0201: Define the <Version> or <VersionPrefix> node explicitly or define the <IsPackable> node with value 'false' (https://dotnet-project-file-analyzers.github.io/rules/Proj0201.html)
Proj0202: Define the <Description> or <PackageDescription> node explicitly or define the <IsPackable> node with value 'false' (https://dotnet-project-file-analyzers.github.io/rules/Proj0202.html)
Proj0203: Define the <Authors> node explicitly or define the <IsPackable> node with value 'false' (https://dotnet-project-file-analyzers.github.io/rules/Proj0203.html)
Proj0204: Define the <PackageTags> node explicitly or define the <IsPackable> node with value 'false' (https://dotnet-project-file-analyzers.github.io/rules/Proj0204.html)
Proj0206: Define the <PackageProjectUrl> node explicitly or define the <IsPackable> node with value 'false' (https://dotnet-project-file-analyzers.github.io/rules/Proj0206.html)
Proj0207: Define the <Copyright> node explicitly or define the <IsPackable> node with value 'false' (https://dotnet-project-file-analyzers.github.io/rules/Proj0207.html)
Proj0208: Define the <PackageReleaseNotes> node explicitly or define the <IsPackable> node with value 'false' (https://dotnet-project-file-analyzers.github.io/rules/Proj0208.html)
Proj0209: Define the <PackageReadmeFile> node explicitly or define the <IsPackable> node with value 'false' (https://dotnet-project-file-analyzers.github.io/rules/Proj0209.html)
Proj0212: Define the <PackageIcon> node explicitly or define the <IsPackable> node with value 'false' (https://dotnet-project-file-analyzers.github.io/rules/Proj0212.html)
Proj0213: Define the <PackageIconUrl> node explicitly or define the <IsPackable> node with value 'false' (https://dotnet-project-file-analyzers.github.io/rules/Proj0213.html)
Proj0210: Define the <PackageLicenseExpression> or <PackageLicenseFile> node explicitly or define the <IsPackable> node with value 'false' (https://dotnet-project-file-analyzers.github.io/rules/Proj0210.html)
Proj0216: Define the <ProductName> node explicitly or define the <IsPackable> node with value 'false' (https://dotnet-project-file-analyzers.github.io/rules/Proj0216.html)
Proj0217: Define the <PackageRequireLicenseAcceptance> node explicitly or define the <IsPackable> node with value 'false' (https://dotnet-project-file-analyzers.github.io/rules/Proj0217.html)
ty line (https://dotnet-project-file-analyzers.github.io/rules/Proj1703.html)
pty line (https://dotnet-project-file-analyzers.github.io/rules/Proj1703.html)
Proj0240: Define the <EnablePackageValidation> node with value 'true' or define the <IsPackable> node with value 'false' or define the <DevelopmentDependency> node with value 'true' (https://dotnet-project-file-analyzers.github.io/rules/Proj0240.html)
Proj0243: Enable SBOM generation with <GenerateSBOM> is 'true' or define the <IsPackable> node with value 'false' (https://dotnet-project-file-analyzers.github.io/rules/Proj0243.html)
Proj0048: Define <LangVersion> with an explicit version number (https://dotnet-project-file-analyzers.github.io/rules/Proj0048.html)
Proj0219: The <IsPackable> node is set to 'true' but the <OutputType> is not 'Library' (https://dotnet-project-file-analyzers.github.io/rules/Proj0219.html)
Proj1300: Remove the project reference to executable ../x.csproj (https://dotnet-project-file-analyzers.github.io/rules/Proj1300.html)

And for 1300 the x.csproj is there because the main project is a benchmark project so maybe not strictly a FP

**Comment by Corniel** [comment]
I think something is not visible here. But the rule states that you sould explicitly set `<IsPackable>` to `false` (or `true`), what you did not do. Or dit I miss something?

**Comment by Corniel** [comment]
@Frulfump Can you confirm that `<IsPackable>` was explictly set to `false`? If not, I''m gonna close it, as then is works like it should.

**Comment by Frulfump** [comment]
> @Frulfump Can you confirm that `<IsPackable>` was explictly set to `false`? If not, I''m gonna close it, as then is works like it should.

@Corniel I don't set the property explicitly to either `false` or `true` but I believe the default value is `true`?

**Comment by Frulfump** [comment]
> > [@Frulfump](https://github.com/Frulfump) Can you confirm that `<IsPackable>` was explictly set to `false`? If not, I''m gonna close it, as then is works like it should.
> 
> [@Corniel](https://github.com/Corniel) I don't set the property explicitly to either `false` or `true` but I believe the default value is `true`?

I tried to verify and found https://learn.microsoft.com/en-us/nuget/reference/msbuild-targets#pack-target-inputs which says the default is true and https://learn.microsoft.com/en-us/dotnet/core/tools/dotnet-pack which says
> Web projects aren't packable.
And the WebSDK explicitly sets it to false https://github.com/dotnet/sdk/blob/main/src/WebSdk/ProjectSystem/Targets/Microsoft.NET.Sdk.Web.ProjectSystem.props#L21

**Comment by Corniel** [comment]
We could decide that webSDK projects are disabled by default, that might be a good choice. But your examples are not using the Web SDK or am I missing something?

We strongly believe that an explict choise should be made for projects being packable or not, and based on the choice a lot of packing data has to be provided.

**Comment by Frulfump** [comment]
Yes WebSDK disabled by default could make sense since it's "known" that it implicitly sets `IsPackable` to `false` since it's an active action taken by the WebSDK developers it could make sense in this case to not force downstream users to set what is already set (maybe that would trigger your rule of not setting something again that is already set?).
Correct I'm not using WebSDK.

Ah ok I understand that your intention is for it to be explicit about `IsPackable`, and that's probably sound as we saw it's not always apparent and it's a design choice you make (or should make) upfront when creating a new project. (Maybe getting some of the default .NET templates to set this property would be good for the ecosystem?)

So maybe this issue should be closed as not planned and split of to a separate WebSDK disabled rule by default issue? (Or maybe adjusted for WebSDK so that it allows the property to be set implicitly but still checks the value to make sure it's `false`?)

**Comment by Corniel** [comment]
@Frulfump Could you please review #781?

**Comment by Frulfump** [comment]
Confirm fixed in 1.12.2
